docs/examples/Transformer Examples.ipynb

```diff
@@ -0,0 +1,563 @@
+   "cell_type": "markdown",
+   "id": "c25444ec",
+   "metadata": {},
+   "source": [
+    "### One-time power flow calculation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 173,
+   "id": "b4e63620",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "------node result------\n",
+      "   id  energized      u_pu             u   u_angle             p             q\n",
+      "0   2          1  0.257969  35599.780770  0.061612  3.193357e+07  1.915647e+09\n",
+      "1   4          1  0.162156  11188.754086  0.579654 -1.000000e+03 -5.000000e+03\n",
+      "2   6          1  0.000001      0.018674  0.502210 -9.999985e+01 -1.000000e+03\n"
+     ]
+    }
+   ],
+   "source": [
+    "# validation (optional)\n",
+    "from power_grid_model.validation import assert_valid_input_data\n",
+    "assert_valid_input_data(input_data=input_data3, calculation_type=CalculationType.power_flow)\n",
+    "\n",
+    "# construction\n",
+    "model3 = PowerGridModel(input_data3)\n",
+    "\n",
+    "# one-time power flow calculation\n",
+    "output_data3 = model3.calculate_power_flow(\n",
+    "    symmetric=True,\n",
+    "    error_tolerance=1e-8,\n",
+    "    max_iterations=20,\n",
+    "    calculation_method=CalculationMethod.newton_raphson)\n",
+    "\n",
+    "# result dataset\n",
+    "print(\"------node result------\")\n",
+    "print(pd.DataFrame(output_data3[\"node\"]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "36f829f1",
+   "metadata": {},
+   "source": [
+    "### One-time State Estimation\n",
+    "\n",
+    "Below we present a simple example of state estimation for a network with a three-winding transformer. \n",
+    "\n",
+    "NOTE: In `power-grid-model`, two-winding transformers belong to `branch3` component type, therefore the `measured_terminal_type` of power sensors should be assigned to `MeasuredTerminalType.branch3_1/_2/_3`.\n",
+    "\n",
+    "For detailed explanation of the arguments, batch calculations and asymmetric calculations, we refer to the [State Estimation Example](./State%20Estimation%20Example.ipynb) and [Asymmetric Calculation Example](./Asymmetric%20Calculation%20Example.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 175,
+   "id": "1ae6c043",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "------node result------\n",
+      "   id  energized   loading           p_1           q_1          i_1  \\\n",
+      "0   3          1  0.334206  1.350446e+06  2.000684e+07  5815.410129   \n",
+      "\n",
+      "            s_1            p_2           q_2          i_2           s_2  \\\n",
+      "0  2.005236e+07 -548726.581018  1.006322e+06  9460.593864  1.146204e+06   \n",
+      "\n",
+      "             p_3           q_3         i_3           s_3  \n",
+      "0 -554680.323429  2.007408e+06  938.913179  2.082632e+06  \n"
```

Comment:
negative `p` is quite unexpected for most people. this probably has to do with the issue for power flow for three-winding transformer, though